src/posix/platform/netif.cpp

```diff
@@ -664,6 +660,9 @@ static void logAddrEvent(bool isAdd, bool isUnicast, struct sockaddr_in6 &addr6,
 
 static void processNetifAddrEvent(otInstance *aInstance, struct nlmsghdr *aNetlinkMessage)
 {
+    constexpr bool kAddAddress    = true;
```

Comment:
I wonder if it's necessary to define these constants? Can we just use `true/false`? 
@superwhd @wgtdkp @bukepo

Reply:
SGTM.